docs/appendix/glossary/board.md

```diff
@@ -2,10 +2,13 @@
 title: Board
 id: board
 full_link: /components/board/
-short_description: An IO board that is, or is connected to, a robot part and used to express low-level electronics functionality such as GPIO, SPI, or I2C.
+short_description: A board is the signal wire hub of a robot that provides access to GPIO pins, SPI or I2C.
 aka:
 ---
 
-An IO board that is, or is connected to, a robot part and used to express low-level electronics functionality such as GPIO, SPI, or I2C.
+A board is the signal wire hub of a robot that provides access to GPIO pins, SPI or I2C.
 
-Examples of boards include Jetson, Raspberry Pi, Numato, or Arduino. Boards like Jetson and Pi run viam-server and can expose the board {{< glossary_tooltip term_id="component" text="component" >}} itself.
+Examples of boards include Jetson, Raspberry Pi, Numato, or Arduino.
+Boards like Jetson and Pi run viam-server and can expose the board {{< glossary_tooltip term_id="component" text="component" >}} itself.
```

Comment:
```suggestion
Boards like Jetson and Pi run `viam-server` and can expose the board {{< glossary_tooltip term_id="component" text="component" >}} itself.
```
What does this sentence even mean though?

Reply:
Happy to remove